ophyd/controls/signal.py

```diff
@@ -285,6 +285,13 @@ def _connected(self, pvname=None, conn=None, pv=None, **kwargs):
             self._ses_logger.debug(msg)
 
     @property
+    def connected(self):
+        if self._write_pv is None:
+            return self._read_pv.connected
+        else:
+            return self._read_pv.connected and self._write_pv.connected
```

Comment:
@tacaswell  has me well trained to replace if/else blocks. Consider replacing this if/else block with:

```python
if self._write_pv:
    return self._read_pv.connected and self._write_pv.connected
return self._read_pv.connected
```